docs/src/differentiable.md

```diff
@@ -9,7 +9,7 @@ The concept of using automatic differentiation to obtain exact gradients through
 There are some projects that explore differentiable molecular simulations, such as [Jax, M.D.](https://github.com/google/jax-md) and [DiffTaichi](https://github.com/yuanming-hu/difftaichi), or you can write your own algorithms in something like [PyTorch](https://pytorch.org).
 However Julia provides a strong suite of autodiff tools, with [Zygote.jl](https://github.com/FluxML/Zygote.jl) allowing source-to-source transformations for much of the language.
 The [differentiable](https://github.com/JuliaMolSim/Molly.jl/tree/differentiable) branch of Molly lets you use the power of Zygote to obtain gradients through molecular simulations.
-It is not yet merged because it is experimental, untested, slow, liable to change, has a slightly different API to the main package, and only works for some parts of the main package.
+It is not yet merged because it is experimental, untested, slow, liable to change, and only works for some parts of the main package.
```

Comment:
The code branch that would be used for Zygote is the one you called `implementation2`?

Reply:
Eventually yes, the out-of-place implementation will work with Zygote, though it doesn't yet. At that point the experimental tag on the differentiable stuff can be taken off. The current `differentiable` branch is an older version of the out-of-place implementation that is very slow but does work with Zygote. So it's a third implementation on an experimental branch that will hopefully be deleted soon.